libs/feature/editor/src/lib/components/record-form/record-form.component.ts

```diff
@@ -8,19 +8,21 @@ import {
   EditorFieldWithValue,
   EditorSectionWithValues,
 } from '../../+state/editor.models'
+import { getGlobalConfig } from '@geonetwork-ui/util/app-config'
+import { UiInputsModule } from '@geonetwork-ui/ui/inputs'
 
 @Component({
   selector: 'gn-ui-record-form',
   templateUrl: './record-form.component.html',
   styleUrls: ['./record-form.component.css'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   standalone: true,
-  imports: [CommonModule, FormFieldComponent, TranslateModule],
+  imports: [CommonModule, FormFieldComponent, TranslateModule, UiInputsModule],
 })
 export class RecordFormComponent {
   constructor(public facade: EditorFacade) {}
 
-  handleFieldValueChange(model: string, newValue: EditorFieldValue) {
+  handleFieldValueChange(model: any, newValue: EditorFieldValue) {
```

Comment:
Why did you need this? The model is a key into a dictionary of values, the key being always a string.

Reply:
I think this is also leftover from v1, when I had changed that function a bit. Will rollback.